packages/transformer/src/IModelExporter.ts

```diff
@@ -870,6 +895,13 @@ export interface ChangedInstanceIdsInitOptions extends ExportChangesOptions {
   iModel: BriefcaseDb;
 }
 
+export interface ExporterInitOptions {
```

Comment:
if you add a type, mark it @beta for now.
Also this type has nothing in it... I agree we should add a new name, but make the properties inline:

```ts
export type ExporterInitOptions = ExportChangesOptions;
```
Later if we add more properties we can do:
```ts
export interface ExporterInitOptions extends ExportChangesOptions {
  newProperty: string;
}
```